pandas/core/generic.py

```diff
@@ -2,9 +2,8 @@
 from __future__ import annotations
 
 import collections
-from datetime import timedelta
+import datetime as dt
```

Comment:
now you'll need to replace `timedelta` (not the function argument) with `dt.timedelta`